### ISSUE TITLE
[Feature] Island Level Based on Designated Valuables Chests

### ISSUE THREAD

**Issue by Bogden** [BentoBoxWorld/Level#220]
With the current level system, islands inevitably end up super gaudy, because players are incentivized to spam whatever blocks are the highest value:

![image](https://user-images.githubusercontent.com/6186474/117228223-81a66a00-adcd-11eb-9d4b-c4a66b6333f0.png)

![image](https://user-images.githubusercontent.com/6186474/117228585-2cb72380-adce-11eb-824c-a349865ed9e2.png)

It might be nice to have a feature where specific blocks could be designated to contribute towards island value without needing to place them. For instance, by sticking a sign on a chest with an `[is level]` tag on it, a server-configured list of blocks could contribute to island level without needing to be placed. By default, I'd suggest the ores/metals: Diamond, Emerald, Gold, Iron, Redstone, Lapis, and maybe even Coal.

**Comment by BONNe** [comment]
There is an option in level addon config that allows to enable cheats content contribution to the island level

**Comment by Bogden** [comment]
That's only an all or nothing config though, right?

**Comment by tastybento** [comment]
Yes. But it was designed to help solve this problem. I'm not sure what designating chests would add. 

Another way to solve this is to use the block limits options. You can only allow counts up to a maximum, even for diamond blocks or gold. Then it'll become a quest to try and work out how to maximize level by using all the diverse blocks they can. It would mean players could max out their level, I must admit.

**Comment by Bogden** [comment]
The idea is to be able to designate blocks in particular, so the super high value stuff that players are tempted to hoard could still be hoarded, but building out the island would still be rewarded without feeling like you _need_ to use diamond blocks everywhere to maximize what you're doing.

With the all or nothing toggle, it doesn't feel great to reward island levels to an endless cobble generator that's just going into a chest, for instance. It's not valuable or hard to get, and it doesn't make the island look bigger or better in any way.

Designating chests is less helpful for sure, just an nice little feature if you wanted to limit the number of chests a player could use to count towards island level, and even potentially hook into it to allow players to increase that number somehow.
